functional-tests/test-bed/src/main/java/org/arquillian/smart/testing/ftest/testbed/project/ProjectBuilder.java

```diff
@@ -127,10 +126,7 @@ private void setCustomMavenInstallation(PomEquippedEmbeddedMaven embeddedMaven)
         if (buildConfigurator.getMavenVersion() != null && !buildConfigurator.getMavenVersion().isEmpty()) {
             setMavenVersion(embeddedMaven);
         } else {
-            final String mvnInstallation = System.getenv("TEST_BED_M2_HOME");
-            if (mvnInstallation != null) {
-                embeddedMaven.useInstallation(new File(mvnInstallation));
-            }
+            embeddedMaven.useDefaultDistribution();
```

Comment:
I would still leave an option to change distribution - this will give us an easy way to test against different versions of maven in e.g. matrix jobs.

Reply:
Good idea with it - I didn't realize some matrix jobs. Currently, different Maven versions are set using the API: https://github.com/arquillian/smart-testing/blob/master/functional-tests/test-bed/src/test/java/org/arquillian/smart/testing/ftest/configuration/DebugModeAndMaven325FunctionalTest.java#L61
On the other hand - these corner cases and supports for older versions should be tested in every build and having another job could just slow down the test suite execution. That's why we have this particular test - to verify backward compatibility.

Reply:
I think having both options - global and per-test are valuable. Do we agree on leaving them then?

Reply:
Yeah, I'm fine with some system property (providing the required Maven version) that would allow us to test it in some matrix jobs.
so +1